pkg/agent/metrics/prometheus.go

```diff
@@ -94,6 +94,30 @@ var (
 		},
 		[]string{"operation"},
 	)
+
+	TotalConnectionsInConnTrackTable = metrics.NewGauge(
+		&metrics.GaugeOpts{
+			Name:           "antrea_agent_conntrack_total_connection_count",
+			Help:           "Number of connections in the conntrack table. This metric gets updated at an interval specified by flowPollInterval, a configMap parameter in antrea-agent.conf.",
+			StabilityLevel: metrics.ALPHA,
+		},
+	)
+
+	TotalAntreaConnectionsInConnTrackTable = metrics.NewGauge(
+		&metrics.GaugeOpts{
+			Name:           "antrea_agent_conntrack_antrea_connection_count",
+			Help:           "Number of connections in the Antrea ZoneID of the conntrack table. This metric gets updated at an interval specified by flowPollInterval, a configMap parameter in antrea-agent.conf.",
```

Comment:
I think the metrics is not updated at an interval but updated realtime?

Reply:
As the scraping interval of Prometheus server may not see the changes in the metrics, I want to be explicit about it being updated by specified interval.

Reply:
it still happens as part of the polling process, but I see your point as we call `Inc` / `Dec` for each inserted / deleted connection, and we don't set the metric value once during polling.

BTW @srikartati do you know why we are not just calling `Set` with `len(filteredConns)`?

Reply:
That is an alternative I thought of but felt that the `Inc/Dec` is a little bit efficient as the number of `Set` calls would be typically higher as we call it every poll interval compared to the case of addition or deletion of the connection.